File: ruby/raindrops/raindrops.rb
class Raindrops
  def self.convert(input)
    output = ""
    output << "Pling" if input % 3 == 0
    output << "Plang" if input % 5 == 0
    output << "Plong" if input % 7 == 0

    output == "" ? input.to_s : output
  end
end

module BookKeeping
  VERSION = 3
end

File: php/raindrops/raindrops.php
<?php

function raindrops($num) {
    $drops = "";
    if ($num % 3 === 0) {
        $drops .= "Pling";
    }
    if ($num % 5 === 0) {
        $drops .= "Plang";
    }
    if($num % 7 === 0) {
        $drops .= "Plong";
    }

    return empty($drops) ? (string)$num : $drops;
}